test-cases/javascript/interactive/multi-step-request-event-listener.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<body>
+    <title>CrawlMaze - Testbed for Web Crawlers</title>
+    <h1> multi-step GET method with EventListener</h1>
```

Comment:
You can add a bit more context just like in test-cases/javascript/interactive/multi-step-request-event-listener-dom.html e.g. `multi-step GET method with EventListener after adding to DOM`